enqueue/fs/0.4/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "ENQUEUE_DSN": "file:/%VAR_DIR%/enqueue"
```

Comment:
I followed the syntax I saw in [chrome's  address bar](https://www.dropbox.com/s/2ajw18yjqp04otu/Screenshot%202017-05-24%2009.47.06.png?dl=0)

If `%VAR_DIR%` is `/home/foo/acme/var` the dsn become `file://home/foo/acme/var/enqueue`